src/components/AboutMe/styles/index.css

```diff
@@ -21,6 +21,41 @@
 	animation-fill-mode: forwards;
 }
 
+.sam-profile-pic {
```

Comment:
Each component should be separated so have it's on css which it's using. Since these are two different components you should extract this and your AboutMeSamTas into a separate folder and place those there. This way our code is not tightly coupled and its easy to differentiate what belongs to who without prepending or post pending your name

so 
AboutMeSamTas(folder)
|_styles(folder)
|   |_ index.css
|_ index.tsx